src/DataAccess/DoctrineDonationRepository.php

```diff
@@ -207,15 +207,11 @@ public function getDonationById( int $id ) {
 	 * @return DoctrineDonation|null
 	 * @throws ORMException
 	 */
-	private function getDoctrineDonationById( int $id ) {
-		/*
-		 * FIXME: only select donations that have
-			 * 'isPublic' => true,
-			 * 'dtDel' => null
-			 * correct status
-		 */
-
-		return $this->entityManager->find( DoctrineDonation::class, $id );
+	public function getDoctrineDonationById( int $id ) {
+		return $this->entityManager->getRepository( DoctrineDonation::class )->findOneBy( [
+			'id' => $id,
+			'dtDel' => null
```

Comment:
This field is about deletion via the backend app right? Or am I confusing it with the cancellation stuff we have in this app?